chore(bump): 3.15.0-alpha.0 (#4517)
Opentrons/opentrons: api/release-notes.md

# Robot OS Changes from 3.14.1 to 3.15.0

For more details about this release, please see the full [technical change log][changelog]

[changelog]: https://github.com/Opentrons/opentrons/blob/edge/CHANGELOG.md

## Python Protocol API V2 Release

> We're happy to announce the full release version 2 of our Python protocol API! To read more, please go to our [documentation](https://docs.opentrons.com/v2/index.html).

Reviewer: ```suggestion
We're happy to announce the full release of version 2 of our Python protocol API! To read more, please go to our [documentation](https://docs.opentrons.com/v2/index.html).
```
sorry

# Robot OS Changes from 3.14.1 to 3.15.0

For more details about this release, please see the full [technical change log][changelog]

[changelog]: https://github.com/Opentrons/opentrons/blob/edge/CHANGELOG.md

## Python Protocol API V2 Release

> We're happy to announce the full release version 2 of our Python protocol API! To read more, please go to our [documentation]: https://docs.opentrons.com/v2/index.html.

Reviewer: ```suggestion
We're happy to announce the full release version 2 of our Python protocol API! To read more, please go to our [documentation](https://docs.opentrons.com/v2/index.html).
```

link formatting

## Thermocycler Release

> We're also happy to announce that software support for our new Thermocycler module is complete!

Reviewer: docs link? product page link?

## Other Changes

> - If you are running an API v2 protocol, you must now specify the version level. Please go to this [link][https://docs.opentrons.com/v2/index.html#metadata-and-version-selection] to read more.

Reviewer: language consistency and hyperlink formatting

```suggestion
- If you are running a Python Protocol API version 2 protocol, you must now specify the version level. Please go to [the documentation](https://docs.opentrons.com/v2/index.html#metadata-and-version-selection) to read more.
```

> - You can now upload API v1 and API v2 protocols without changing a feature flag.

Reviewer: ```suggestion
- You can now upload Python Protocol API version 1 and version 2 protocols without changing a feature flag.
```

language consistency

> - You can specify starting volumes for Thermocycler temperature commands. Read [here][] for more information.

Reviewer: ```suggestion
- You can specify starting volumes for Thermocycler Module temperature commands. Please read the documentation on [setting block temperature](http://sandbox.docs.opentrons.com/edge/v2/new_modules.html#block-temperature-control) and [executing temperature profiles](http://sandbox.docs.opentrons.com/edge/v2/new_modules.html#thermocycler-module-profiles) for more information.
```
language consistency, set the links

## Bug Fixes
- Reduce logging noise

> - Fix calibrate to bottom feature in API v2 for tipracks

Reviewer: ```suggestion
- The robot will now always calibrate to the top of tipracks even if the calibrate to bottom advanced setting is set ([#4417](https://github.com/Opentrons/opentrons/issues/4417)). Thanks to Theo Sanderson (GitHub: @theosanderson) for the issue documentation and contribution of the fix!
```

> - Fix module behavior upon protocol cancel

Reviewer: let's be a little more specific here - what behavior did it fix? what is the new behavior?

Also let's add 

- Use a more efficient way to check whether a motor is moving in a given command, resulting in about a 10% speedup in protocol simulation for Protocol API Version 1 protocols and a 5% speedup for Protocol API Version 2 protocols ([#4482](https://github.com/opentrons/opentrons/issues/4482)). Thanks to Robert Atkinson (GitHub: @rgatkinson) for discovering the issue and suggesting a fix!

> ## Known Issues API v1

Reviewer: ```suggestion
## Known Issues (Python Protocol API version 1)
```
